install_scripts/fuseki.sh

```diff
@@ -8,26 +8,19 @@ fi
 
 if [ ! -d $FUSEKI_HOME ]; then
   mkdir $FUSEKI_HOME
+  chown -hR tomcat7:tomcat7 $FUSEKI_HOME
 fi
 
-if [ ! -f "$DOWNLOAD_DIR/jena-fuseki1-$FUSEKI_VERSION-distribution.tar.gz" ]; then
+if [ ! -f "$DOWNLOAD_DIR/apache-jena-fuseki-"$FUSEKI_VERSION".tar.gz" ]; then
   echo -n "Downloading Fuseki..."
-  wget -q -O "$DOWNLOAD_DIR/jena-fuseki1-$FUSEKI_VERSION-distribution.tar.gz" "http://www.apache.org/dist/jena/binaries/jena-fuseki1-"$FUSEKI_VERSION"-distribution.tar.gz"
+  wget -q -O "$DOWNLOAD_DIR/apache-jena-fuseki-"$FUSEKI_VERSION".tar.gz" "http://www.apache.org/dist/jena/binaries/apache-jena-fuseki-"$FUSEKI_VERSION".tar.gz"
   echo " done"
 fi
 
 cd /tmp
-cp "$DOWNLOAD_DIR/jena-fuseki1-$FUSEKI_VERSION-distribution.tar.gz" /tmp
-tar -xzvf jena-fuseki1-"$FUSEKI_VERSION"-distribution.tar.gz
-cd jena-fuseki1-"$FUSEKI_VERSION"
-mv -v * $FUSEKI_HOME
-chown -hR tomcat7:tomcat7 $FUSEKI_HOME
-
-mkdir "$FUSEKI_HOME/test_data"
-ln -s $FUSEKI_HOME/fuseki /etc/init.d
-echo "FUSEKI_HOME=\"$FUSEKI_HOME\"" > /etc/default/fuseki
-echo "FUSEKI_ARGS=\"--update --loc=$FUSEKI_HOME/test_data /test\"" >> /etc/default/fuseki
```

Comment:
In moving to the `war` deployment, it looks like we lost these `FUSEKI_ARGS`.
This PR should ensure all three of these things:
1) the triplestore is update-able
2) the triplestore persistence is at `$FUSEKI_HOME/test_data`
3) the initial dataset is exposed at `/test`